.gitignore

```diff
@@ -1,6 +1,6 @@
 /.idea/
 /.vscode/
 
-/api/vendor/
+/vendor/
```

Comment:
Could you give some additional info, why have you moved `vendor` to root project directory?

Reply:
`vendor/` moved to root just because of Gopkg.toml / Gopkg.lock moved to root.
Gopkg moved to root to properly add dependencies for `/utils/pdeathsig` package. `/api/Gopkg.toml` can't see `/utils/...` packages.
As a side effect, it actually fixed a problem of "key dependencies" versions not being fixed in downstream projects. Thanks to that we can remove hacks in them.